demisto_sdk/commands/common/schemas/playbook.yml

```diff
@@ -311,6 +319,23 @@ schema;input_schema:
     playbookInputQuery:
       type: any
 
+# playbook->inputSections
+schema;inputsSection_schema:
+  type: map
+  mapping:
+    name:
+      type: str
+      required: true
+    id:
+      type: str
```

Comment:
This is optional I understand? 
I think we need to make sure the ID will not be exported when exporting the playbook yml

Reply:
It shouldn't be exported from XSOAR when we download it, I think we can remove this, but I'll ensure.